Library/Homebrew/download_strategy.rb

```diff
@@ -1277,7 +1277,7 @@ def update(timeout: nil)
 class BazaarDownloadStrategy < VCSDownloadStrategy
   def initialize(url, name, version, **meta)
     super
-    @url.sub!(%r{^bzr://}, "")
+    @url = @url.sub(%r{^bzr://}, "")
```

Comment:
cc @issyl0 this should probably be `@url = @url.delete_prefix("bzr://")`. It looks like we would need to disable `SafeMultiline` in `Performance/DeletePrefix` for Rubocop to flag this (IME 99% a ruby regex uses `^`/`$` it should be using `\A`/`/z`).